oleesir/authors-haven-api
# #168675532 Email verification

#### What does this PR do?
This PR implements email verification endpoint
#### Description of Task to be completed?
 Implements the email verification endpoint
#### How should this be manually tested?
- git fetch on your local machine and checkout to ft-email-verification-168675532 API branch
- Run "npm run dev" to start the development server
- Open postman or any other API client
- Make API call to ```POST/api/v1/auth/signup```
- Make API call to ```GET/api/v1 /auth/verification?token[yourtoken]&email=[youremail]``` to verify the email
#### What are the relevant pivotal tracker stories?
[#168675532](https://www.pivotaltracker.com/story/show/168675532)
#### Any background context you want to add?
N/A
#### Screenshots
N/A
